packages/components/stepper/src/js/step-form/step-form.controller.js

```diff
@@ -62,18 +62,34 @@ export default class StepFormController {
     }
   }
 
+  getNodeIndex() {
+    let i = 0;
+    let item = this.$element[0];
+
+    // Return DOM node index
+    while (item.previousSibling) {
+      item = item.previousSibling;
+      if (item.nodeType === 1) {
```

Comment:
What's a `nodeType` whose value is `1` ? Either add a comment or make it explicit

i.e.:
````javascript
if (node.type === TYPES.Datagrid) { // :D
````

Reply:
Used `Node.ELEMENT_NODE`